sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlterTableDropPartitionSuiteBase.scala

```diff
@@ -21,6 +21,20 @@ import org.apache.spark.sql.{AnalysisException, QueryTest}
 import org.apache.spark.sql.catalyst.analysis.NoSuchPartitionsException
 import org.apache.spark.sql.internal.SQLConf
 
+/**
+ * This base suite contains unified tests for the `ALTER TABLE .. DROP PARTITION` command that
+ * check V1 and V2 table catalogs. The tests that cannot run for all supported catalogs are
+ * located in more specific test suites:
+ *
+ *   - V2 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v2.AlterTableDropPartitionSuite]]
+ *   - V1 table catalog tests:
+ *     [[org.apache.spark.sql.execution.command.v1.AlterTableDropPartitionSuiteBase]]
```

Comment:
`org.apache.spark.execution.command.AlterTableDropPartitionSuiteBase` ?

Reply:
No, I think the ref is correct, see https://github.com/apache/spark/blob/29cca68e9e55fae8389378de6f30d0dfa7a74010/sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/AlterTableDropPartitionSuite.scala#L23

Reply:
My bad~